src/site/content/en/blog/gamepad/index.md

```diff
@@ -0,0 +1,348 @@
+
+The schematic overview below, taken
+[straight from the spec](https://w3c.github.io/gamepad/#fig-visual-representation-of-a-standard-gamepad-layout:~:text=Figure%201%20Visual%20representation%20of%20a%20standard%20gamepad%20layout.),
+shows the mapping and the arrangement of the buttons and axes on a generic gamepad.
+
+<figure class="w-figure">
+  <img src="gamepad.svg"
+       alt="Schematic overview of the button and axes mappings of a common gamepad."
+  >
+  <figcaption class="w-figcaption">
+    Visual representation of a standard gamepad layout
+    (<a href="https://w3c.github.io/gamepad/#fig-visual-representation-of-a-standard-gamepad-layout:~:text=Figure%201%20Visual%20representation%20of%20a%20standard%20gamepad%20layout.">Source</a>).
+  </figcaption>
+</figure>
+
+### Being notified when a gamepad gets connected
+
+To learn when a gamepad is connected, listen for the `gamepadconnected`
+event that triggers on the `window` object.
+When the user connects a gamepad, which can either happen via USB or via Bluetooth,
+a `GamepadEvent` is fired that has the gamepad's details in an aptly named `gamepad` property.
+Below, you can see an example from an Xbox 360 controller that I had lying around
+(yes, I am into retro gaming).
+
+```js/3-12
+window.addEventListener('gamepadconnected', (event) => {
+  console.log('✅ 🎮 A gamepad was connected:', event.gamepad);
+  /*
+    gamepad: Gamepad
+    axes: (4) [0, 0, 0, 0]
+    buttons: (17) [GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton]
+    connected: true
+    id: "Xbox 360 Controller (STANDARD GAMEPAD Vendor: 045e Product: 028e)"
+    index: 0
+    mapping: "standard"
+    timestamp: 6563054.284999998
+    vibrationActuator: GamepadHapticActuator {type: "dual-rumble"}
+  */
+});
+```
+
+### Being notified when a gamepad gets disconnected
+
+Being notified of gamepad disconnects happens analogously to the way connections are detected.
+This time the app listens for the `gamepaddisconnect` event.
+Note how in the example below `connected` is now `false`
+when I unplug the Xbox 360 controller.
+
+```js/6
+window.addEventListener('gamepadconnected', (event) => {
+  console.log('❌ 🎮 A gamepad was disconnected:', event.gamepad);
+  /*
+    gamepad: Gamepad
+    axes: (4) [0, 0, 0, 0]
+    buttons: (17) [GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton]
+    connected: false
+    id: "Xbox 360 Controller (STANDARD GAMEPAD Vendor: 045e Product: 028e)"
+    index: 0
+    mapping: "standard"
+    timestamp: 6563054.284999998
+    vibrationActuator: null
```

Comment:
No more `vibrationActuator`?

Reply:
I don't know why it's forgotten, it reports `null`.